src/serialization/string_utils.cpp

```diff
@@ -828,9 +831,13 @@ std::pair< int, int > parse_range(std::string const &str)
 	const std::string::const_iterator dash = std::find(str.begin(), str.end(), '-');
 	const std::string a(str.begin(), dash);
 	const std::string b = dash != str.end() ? std::string(dash + 1, str.end()) : a;
-	std::pair<int,int> res(std::stoi(a), std::stoi(b));
-	if (res.second < res.first)
-		res.second = res.first;
+	std::pair<int,int> res;
```

Comment:
I'm not quite sure if this will be zero-initialized; might be better to explicitly initialize it to `{0,0}`.